orderbook/src/fee.rs

```diff
@@ -0,0 +1,178 @@
+use std::collections::HashMap;
+
+use anyhow::Result;
+use chrono::{DateTime, Duration, Utc};
+use primitive_types::{H160, U256};
+use std::sync::Mutex;
+
+use crate::price_estimate::PriceEstimating;
+use gas_estimation::GasPriceEstimating;
+
+type Measurement = (U256, DateTime<Utc>);
```

Comment:
Like I said in the other comment this could be SystemTime instead of DateTime too. For the http api we would then convert it to DateTime in order to format it correctly but here it doesn't need to be. I don't know if this is better though, so  no need to change if you like DateTime.

Reply:
For writing it to the DB it might be nice to be denoted in DateTime<Utc> rather than system time (although system time seems to be UTC too). It's less conversion work to leave it as is.